c++/src/kj/common.h

```diff
@@ -1995,6 +1995,12 @@ inline constexpr ArrayPtr<T> arrayPtr(T* begin KJ_LIFETIMEBOUND, T* end KJ_LIFET
   return ArrayPtr<T>(begin, end);
 }
 
+template <typename T, size_t Size>
```

Comment:
Not worth a PR to fix, but FWIW, I've always gone by the rule that type names start with a capital letter, values start with a lower-case letter -- including for template parameters. So `Size` here should actually be `size`.

Reply:
no worries: https://github.com/capnproto/capnproto/pull/2010